ssh/tailssh/tailssh.go

```diff
@@ -410,9 +411,6 @@ func (srv *server) newConn() (*conn, error) {
 		Version:              "Tailscale",
 		ServerConfigCallback: c.ServerConfig,
 
-		NoClientAuthHandler: c.NoClientAuthCallback,
```

Comment:
We no longer use gliderlabs' authentication abstraction, and instead specify our authentication callback [directly on the gossh.ServerConfig](https://github.com/tailscale/tailscale/pull/14803/files#diff-2c1da25297e2920b9b19163586874de638c004ef617832bb74d154f21d2123daR71).